web/data/data.go

```diff
@@ -111,10 +113,19 @@ func createDoc(c echo.Context) error {
 		return err
 	}
 
+	if err := hooks.Run(instance, realtime.EventCreate, &doc, nil); err != nil {
```

Comment:
Why do we call this from `web/data` and not `pkg/couchdb`? I would have thought that in `pkg/couchdb`, it would have been more solid if later we had an hook on a doctype that can be modified from another `web/something` API.